cli.js

```diff
@@ -59,61 +58,33 @@ const cli = meow(chalk`
   $ webpack-serve ./webpack.config.js --no-reload
   $ webpack-serve --config ./webpack.config.js --port 1337
   $ webpack-serve # config can be omitted for webpack v4+ only
-`, {
-  flags: {
-    require: {
-      alias: 'r',
-      type: 'string'
-    }
+`,
+  {
+    flags: {
+      require: {
+        alias: 'r',
+        type: 'string',
+      },
+    },
   }
-});
+);
 
 const flags = Object.assign({}, cli.flags);
 
-if (flags.require) {
```

Comment:
Why did you remove this option?

Reply:
it's handled by `config-loader` downstream